person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name = 'Unknown', age = 0, parent_permission: true)
```

Comment:
- [ ] As mentioned in the project requirements only `parent_permission` and ` name` have a default value, `age` should not have a default value. Kindly fix that 🙃 🙃 
- [OPTIONAL] Instead of creating the `generate_id` method you can do it directly in `intialize` like 👉🏾  `@id = Random.rand(1..1000)` 😁